eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,15 @@
+<!-- Prebuilt baseline changes must be reviewed by @dotnet/source-build-internal -->
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <!-- TODO: Ignore needed until https://github.com/dotnet/source-build-reference-packages/pull/500 flows in. -->
```

Comment:
@MichaelSimons do you want to fix it in this PR? or will it take time?

Reply:
The fix should have flown in with https://github.com/dotnet/templating/pull/5778 so we can removed this todo and the exclusion now before merging.